#! /bin/sh /usr/share/dpatch/dpatch-run
## 20_fix_gimpprint_menu_entry.dpatch by  <till@laptoptill>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Let "Print" entry appear at a nicer place in the "Filr" menu of GIMP

@DPATCH@
diff -urNad gutenprint-5.0.0~/src/gimp2/print-print.c gutenprint-5.0.0/src/gimp2/print-print.c
--- gutenprint-5.0.0~/src/gimp2/print-print.c	2006-07-04 04:57:59.000000000 +0200
+++ gutenprint-5.0.0/src/gimp2/print-print.c	2006-09-13 10:36:32.000000000 +0200
@@ -43,7 +43,7 @@
 			  (BAD_CONST_CHAR) copy,
 			  (BAD_CONST_CHAR) VERSION " - " RELEASE_DATE,
 			  /* Do not translate the prefix "<Image>" */
-			  (BAD_CONST_CHAR) N_("<Image>/File/Print..."),
+			  (BAD_CONST_CHAR) N_("<Image>/File/Send/Print..."),
 			  (BAD_CONST_CHAR) types,
 			  GIMP_PLUGIN,
 			  n_args, 0,
